src/main/java/com/toptrumps/cli/CommandLineController.java

```diff
@@ -114,63 +220,31 @@ private void showRoundResult(RoundOutcome outcome){
         System.out.println(outcomeMessage);
     }
 
-    private void showRemovedPlayers(ArrayList<Player> removedPlayers){
+    private void showRemovedPlayers(List<Player> removedPlayers) {
         String removedPlayersString = "";
-        if(removedPlayers.size()==1){
+        if (removedPlayers.size() == 1) {
             removedPlayersString += removedPlayers.get(0).getName() + " has been removed from the game";
-        }else{
-            Player lastPlayerInList = removedPlayers.get(removedPlayers.size()-1);
-            Player secondLastPlayerInList = removedPlayers.get(removedPlayers.size()-2);
-            for(Player player: removedPlayers){
-                if(player == lastPlayerInList){
+        } else {
+            Player lastPlayerInList = removedPlayers.get(removedPlayers.size() - 1);
+            Player secondLastPlayerInList = removedPlayers.get(removedPlayers.size() - 2);
+            for (Player player : removedPlayers) {
+                if (player == lastPlayerInList) {
                     removedPlayersString += " and " + player.getName() + " have been removed from the game";
-                }else if(player == secondLastPlayerInList){
+                } else if (player == secondLastPlayerInList) {
                     removedPlayersString += player.getName();
-                }else{
+                } else {
                     removedPlayersString += player.getName() + ", ";
                 }
             }
         }
         System.out.println(removedPlayersString);
     }
 
-    @Override
-    public void onGameOver(Player winner) {
-        String message = String.format("\nGAME OVER, %s won\n\n", winner.getName());
-        System.out.println(message);
-        scanner.nextLine(); //clear the scanner ready for new game selection
-        newGameHandler();
-    }
-
-    private void numberOfHumanPlayerCards(int cardsRemaining ) {
```

Comment:
Why was this deleted? It was kinda useful.

Reply:
```newGameHandler()``` has been refactored and now is part of ```start()```; I've also restored its call inside ```onGameOver```. I suggest we incorporate an option to simply **quit** the game.